network/local.go

```diff
@@ -26,16 +29,18 @@ func NewLocalRouterWithManager(ctx *LocalManager, sid *ServerIdentity) (*Router,
 	return NewRouter(sid, h), nil
 }
 
-// LocalManager keeps reference to all opened local connections
-// It also keeps tracks of who is "listening", so it's possible to mimics
+// LocalManager keeps a reference to all opened local connections.
+// It also keeps track of who is "listening", so it's possible to mimic
 // Conn & Listener.
 type LocalManager struct {
 	// queues maps a remote endpoint to its packet queue. It's the main
-	//stucture used to communicate.
+	// stucture used to communicate.
 	queues map[endpoint]*connQueue
 	sync.Mutex
+	// The listening-functions used when a new connection-request arrives.
 	listening map[Address]func(Conn)
 
+	// Unique ID of this Local Manager.
```

Comment:
No it's not. It's more like a counter that we increment for each connection it creates/handle. So maybe rename the field to `counter` or stg.